draft-ietf-rtcweb-jsep.xml

```diff
@@ -2324,11 +2359,107 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
+                  enable them on the associated payload types.</t>
+                <t>For each specified RTCP feedback mechanism that is supported by the local implementation,
+                  enable them on the associated payload types.</t>
+                <t>For any specified "b=AS" or "b=TIAS" bandwidth value, record this value
+                  as the maximum value that the remote side wishes to receive for this media section.
+                  [TODO: convert AS to TIAS].</t>
+                <t>[TODO: handling of CN, telephone-event, "red"]</t>
+                <t>If the media section if of type audio:
+                  <list style="symbols">
+                    <t>For any specified "ptime" value, configure the avaialble payload types to
+                      use the specified packet size.</t>
+                  </list>
+                </t>
+              </list>
+            </t>
+          </list>
+        </t>
+        <t>Finally, if this description is of type "pranswer" or "answer", follow
+        the processing defined in the <xref target="sec.applying-an-answer"/>
+        section below.</t>
       </section>
 
       <section title="Applying an Answer" anchor="sec.applying-an-answer">
-        <t>TODO</t>
+        <t>In addition to the steps mentioned above for processing a local or
+          remote description, the following steps are performed when processing
+          a description of type "pranswer" or "answer".</t>
+        <t>For each media section, the following steps MUST be performed:
+          <list style="symbols">
+            <t>If the media section has been rejected (i.e. port is set to zero in
+               the answer), stop any reception or transmission of media for this
+               section, and discard any associated ICE components. [TODO: ref]</t>
+            <t>If the remote DTLS fingerprint has been changed, tear down the existing
+              DTLS connection.</t>
+            <t>If no valid DTLS connection exists, prepare to start a DTLS connection,
+              using the specified roles and fingerprints, on any underlying ICE components,
+              once they are active.</t>
+            <t>If the media section proto value indicates use of RTP:
+              <list style="symbols">
+                <t>If the media section has RTCP mux enabled, discard any RTCP component,
+                   and begin or continue muxing RTCP over the RTP component, as specified in
+                   <xref target="RFC5761"/>, Section X.Y. Otherwise, transmit RTCP over
+                   the RTCP component; if no RTCP component exists, because RTCP mux
+                   was previously enabled, this MUST result in an error. [TODO: ref]</t>
+                <t>If the media section has reduced-size RTCP enabled, configure the
+                  RTCP transmission for this media section to use reduced-size RTCP,
+                  as specified in [TODO: ref]</t>
```

Comment:
This is covered in the semantics validation section above, but it was buried a bit, so I expanded it to make it more obvious.